### PR TITLE
Introduce geolocation functionality

### PR DESCRIPTION
**Pull request by akfernun**

### CODE REVIEW

File: lib/importers/test_data_importer.rb [hunk 1]
```diff
@@ -3,7 +3,7 @@ module TelephoneNumber
   require 'httparty'
   require 'yaml'
 
-  class TestDataGenerator
+  class TestDataImporter
```
[the remark this answers is not in the document]
Comment (akfernun) [reply lib/importers/test_data_importer.rb]: Yep. Fixed

File: lib/telephone_number/class_methods.rb [hunk 2]
```diff
@@ -34,7 +34,7 @@ def detect_country(number)
 
     # generates binary file from xml that user gives us
     def generate_override_file(file)
-      DataImporter.new(file, override: true).import!
+      PhoneDataImporter.new(file, override: true).import!
```
[the remark this answers is not in the document]
Comment (akfernun) [reply lib/telephone_number/class_methods.rb]: Yep. Fixed

File: .ruby-version [hunk 3]
```diff
@@ -1 +1 @@
-2.2.5
+2.3.1
```
Comment (akfernun) [inline .ruby-version]: remove this.

Comment (trouta23) [reply .ruby-version]: Remove this?

Comment (akfernun) [reply .ruby-version]: Sorry, I don't want to remove the file, but I do want to revert the changes. This was a local development thing that shouldn't be committed

Comment (trouta23) [reply .ruby-version]: Ah gotcha.

**Pull request merged**